src/components/tooltip/tooltip.test.jsx

```diff
@@ -18,6 +18,24 @@ describe('tooltip', () => {
     expect(document.body).toMatchSnapshot();
   });
 
+  it('errors if the tooltip is not describing any elements', () => {
+    /* eslint-disable no-console */
+    const originalConsoleError = console.error;
```

Comment:
Pro-tip: access the window instead of the global console to bypass the linter! (This is a secret)

```suggestion
    const originalConsoleError = window.console.error;
```

Reply:
1d149d67